tutorials/applications/Shadow estimation for VQS.ipynb

```diff
@@ -0,0 +1,1460 @@
+{
```

Comment:
<p>Great notebook. Would it be possible to mention the total number of measurements necessary for this VQS procedure on H_2 ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/252/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='84'/>